packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -154,6 +150,18 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
     this._button.option('disabled', state);
   }
 
+  _updateAria(value: string | null): void {
```

Comment:
I'd like to explain what we reference to here
```suggestion
  _updateAria(emptyViewId: string | null): void {
```

Reply:
[ef44cc43f5326a35f5bd9b69d6ee254a85950752]